packages/@aws-cdk/aws-servicecatalogappregistry/lib/application.ts

```diff
@@ -34,6 +34,12 @@ export interface IApplication extends cdk.IResource {
    */
   readonly applicationName?: string;
 
+  /**
+   * Application manager URL for the Application.
+   * @attribute
+   */
+  readonly applicationManagerUrl?: cdk.CfnOutput;
+
```

Comment:
Don't put it on `IApplication`. By its very nature it's optional, which is not very useful.

More useful to put it *definitely* on `Application`.

Reply:
Addressed in the new revision.